### PR TITLE
horizon-eda: init at 1.1.1

### PR DESCRIPTION
**Pull request by yrashk**
Mostly based on #56497 by @Luz

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by emilazy — CHANGES_REQUESTED** [review]
Needs a rebase, but builds and runs for me afterwards; thanks for working on this!

Please use 2 spaces for indentation rather than the current mix of hard tabs and 2-or-8 space indentation. I've left some more specific comments.

**Comment by yrashk** [comment]
@emilazy I believe I addressed your feedback in an updated commit.

**Review by emilazy — CHANGES_REQUESTED** [review]
Thanks for addressing these! A couple more formatting nitpicks, but other than that this looks good to go to me.

The build failed on 64-bit ARM due to freeimage's vendored libpng; I tried to fix it but got lost in the weeds. It's already tracked in #77653, so I opened #86694 to mark freeimage as broken on AArch64 for now.

**Comment by yrashk** [comment]
@emilazy addressed your indentation remarks

**Review by emilazy — APPROVED** [review]

**Comment by emilazy** [comment]
Looks good, thanks!

File: pkgs/applications/science/electronics/horizon-eda/default.nix [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    libyamlcpp
+    libzip
+    opencascade
+    podofo
+    python3
+    sqlite
+    zeromq
+  ];
+
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/horizon-eda/default.nix:54]:
```suggestion
  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
```

File: pkgs/applications/science/electronics/horizon-eda/default.nix [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/horizon-eda/default.nix:4]:
```suggestion
, pkg-config
```

File: pkgs/applications/science/electronics/horizon-eda/default.nix [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/horizon-eda/default.nix:2]:
```suggestion
{ stdenv
```

File: pkgs/applications/science/electronics/horizon-eda/default.nix [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+  NIX_CFLAGS_COMPILE = "-I${opencascade}/include/oce";
+
+  installPhase = ''
+    make install INSTALL=${coreutils}/bin/install DESTDIR=$out PREFIX=
+  '';
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
```
Comment (SuperSandro2000) [inline pkgs/applications/science/electronics/horizon-eda/default.nix:64]:
```suggestion
  meta = with stdenv.lib; {
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Already newer version in nixpkgs.

**Pull request closed without merging**